### PR TITLE
feat: liikunta theming part 1

### PR DESCRIPTION
**Pull request by melniiv**
## Description
- Front page search component transparent bg
- Footer dark theme in sports
- Footer categories removed in sports
- Set all the theme color variables
- Fixed hcrc variables and added additional to style article search and article page

NOT DONE:
Waiting of styles for the tag components for all 3 apps, as we now have to many style variations. Out of scope of this PR, separate PR is needed

### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-94-liikunta-324-theme.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-94-liikunta-324-theme.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-94-liikunta-324-theme.dev.hel.ninja** :rocket::rocket::rocket:

File: apps/sports-helsinki/src/domain/event/eventKeywords/EventKeywords.tsx [hunk 1]
```diff
@@ -76,82 +77,92 @@ const EventKeywords: React.FC<Props> = ({
...
+              color="engelLight50"
+              blackOnMobile={blackOnMobile}
+              hideOnMobile={hideKeywordsOnMobile}
+              key={second.id}
+              keyword={second.name}
+              onClick={
+                withActions ? handleClick('text', second.name) : undefined
+              }
+            />
+          )}
```
Comment (nikomakela) [inline apps/sports-helsinki/src/domain/event/eventKeywords/EventKeywords.tsx:140]: I'm again confused with this - like I think I was earlier too. Would it make more sense to collect a base component that is shared in every condition (or at least in many) and then just override the values that changes?

```
const base = {
color: "engelLight50",
blackOnMobile: blackOnMobile,
// ...
}

<KeywordTag {...base} color='overriddencolor' key={second.id} keyword={second.name} />
```

**Review by nikomakela — APPROVED** [review]
The EventKeywords looks too complicated: It's hard to understand whats really rendered. The rest is golden.

File: apps/sports-helsinki/src/domain/event/eventKeywords/EventKeywords.tsx (hunk 1 above)
Comment (nikomakela) [reply apps/sports-helsinki/src/domain/event/eventKeywords/EventKeywords.tsx:140]: Maybe an extending KeywordTag in the same file with some fixed props would help?

```
      {today && (
        <KeywordTag
          color="engelLight50"
          keyword={t('categories.labelToday')}
          onClick={
            withActions ? handleClick('dateType', DATE_TYPES.TODAY) : undefined
          }
        />
      )}
```

would become 

```
  const TodayKeyword = () => (
    <KeywordTag
      color="engelLight50"
      keyword={t('categories.labelToday')}
      onClick={
        withActions ? handleClick('dateType', DATE_TYPES.TODAY) : undefined
      }
    />
  );
  ``` 
  
  etc

**Pull request merged**